src/editor/blocks/posts-inserter/utils.js

```diff
@@ -60,9 +60,35 @@ const getExcerptBlockTemplate = ( post, { excerptLength, textFontSize, textColor
 	return [ 'core/paragraph', assignFontSize( textFontSize, attributes ) ];
 };
 
+const getAuthorBlockTemplate = ( post, { textFontSize, textColor } ) => {
+	if (
+		Array.isArray( post.newspack_author_info ) &&
+		post.newspack_author_info.length &&
+		post.newspack_author_info[ 0 ].display_name
+	) {
+		return [
+			'core/paragraph',
+			assignFontSize( textFontSize, {
+				content: 'By ' + post.newspack_author_info[ 0 ].display_name,
```

Comment:
i18n missing – `'By '` should be translatable.

Reply:
Good catch! Fixed in 2e02592.